### PR TITLE
1746 meeting api support merge contacts with meetings

### LINKED ISSUE #1746: Meeting API - support merge contacts with meetings
**Overview of the feature request and the problem you're looking to solve**

**Describe the solution you'd like**

**Additional context**

Thank you for your feature request!  We strive to make the product better every day, and your input helps!

### PR DESCRIPTION
**Pull request by gabriel0003**
## Changes

What types of changes does your code introduce?  _Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Build related changes
- [ ] Documentation Update (if none of the other choices apply)
- [ ] Other (please describe below)

### CODE REVIEW

File: packages/server/customer-os-api/graph/resolver/meeting.resolvers_it_test.go [hunk 1]
```diff
@@ -119,6 +120,80 @@ func TestMutationResolver_Meeting(t *testing.T) {
 	require.Equal(t, "test-recording-id", meeting.Meeting_Update.Recording)
 }
 
+func TestMutationResolver_MergeContactsWithMeetings(t *testing.T) {
+	ctx := context.TODO()
+	defer tearDownTestCase(ctx)(t)
+	neo4jt.CreateTenant(ctx, driver, tenantName)
+	neo4jt.CreateDefaultUserWithId(ctx, driver, tenantName, "test_user_id")
+	neo4jt.CreateContactWithId(ctx, driver, tenantName, "test_contact_id_1", entity.ContactEntity{
```
Comment (edifirut) [inline packages/server/customer-os-api/graph/resolver/meeting.resolvers_it_test.go]: please extract all the ids as a vars. it's easy to track in code than the actual string
contactId1 := "test_contact_id_1"

File: packages/server/customer-os-api/repository/contact_repository.go [hunk 2]
```diff
@@ -717,7 +717,26 @@ func (r *contactRepository) MergeContactRelationsInTx(ctx context.Context, tx ne
...
+		"			SET	rel.merged=true", params); err != nil {
+		return err
+	}
+	if _, err := tx.Run(ctx, matchQuery+
+		" WITH primary, merged "+
+		" MATCH (merged)<-[rel:ATTENDED_BY]-(m:Meeting) "+
+		" MERGE (primary)<-[newRel:ATTENDED_BY]-(m) "+
+		" ON CREATE SET newRel.mergedFrom = $mergedContactId, "+
+		"				newRel.createdAt = $now, "+
+		"				newRel.type = rel.type "+
```
Comment (alexopenline) [inline packages/server/customer-os-api/repository/contact_repository.go:736]: is type field belongs to property ATTENDED_BY / CREATED_BY ?
if not to be excluded in both cypher queries

Comment (edifirut) [reply packages/server/customer-os-api/repository/contact_repository.go:736]: type property from relation is going to be taken out by me in a new PR ( tomorrow ). I will do the changes for this on my side

File: packages/server/customer-os-api/graph/resolver/meeting.resolvers_it_test.go (hunk 1 above)
Comment (gabriel0003) [reply packages/server/customer-os-api/graph/resolver/meeting.resolvers_it_test.go]: done

**Review by edifirut — APPROVED** [review]

**Pull request merged**